packages/astro/e2e/test-utils.js

```diff
@@ -18,7 +18,7 @@ export function testFactory(inlineConfig) {
 
 	const test = testBase.extend({
 		astro: async ({}, use) => {
-			fixture = await loadFixture(inlineConfig);
+			fixture = fixture || await loadFixture(inlineConfig);
```

Comment:
That's it? Amazing!

Reply:
That's the one!  Test suites using `beforeAll` for firing up the dev server also helps speed up suites a bit